tests/integration_tests/performance/test_versioned_serialization_benchmark.py

```diff
@@ -115,7 +114,7 @@ def test_serialization_benchmark():
         mean = estimates["mean"]["point_estimate"] / NSEC_IN_MSEC
         logger.info("Mean: %f", mean)
 
-        res = _check_statistics(directory, mean)
+        res = _check_statistics(directory, round(mean, 3))
```

Comment:
Not sure what the round is for... to eliminate noise in measurements? If so, there is also `pytest.approx` https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest-approx